src/metabase/driver.clj

```diff
@@ -304,7 +304,7 @@
         java.math.BigInteger            :BigIntegerField
         java.sql.Date                   :DateField
         java.sql.Timestamp              :DateTimeField
-        java.util.Date                  :DateField
+        java.util.Date                  :DateTimeField
```

Comment:
No. There's a theoretical difference between the two but AFAIK there aren't any places where we *actually* treat `:DateField` differently than we do `:DateTimeField`. Similar to `:CharField` vs. `:TextField` -- they're effectively synonyms at this point, a carryover from the old Django days. So I was curious if you saw a place where this actually changed behaviour